bin/vault/usage.py

```diff
@@ -87,15 +87,28 @@ def _parser_factory():
                 nargs="?",
                 const="all",
                 help=_actions[action].view_help)
+
     sub_parser.add_argument(
                 "--absolute",
                 action="store_true",
                 help=_absolute_help
     )
-    sub_parser.add_argument(
+
+    
+    files_mutually_exclusive_group = sub_parser.add_mutually_exclusive_group()
+
+    files_mutually_exclusive_group.add_argument(
+            "--fofn",
```

Comment:
It looks like a lot of these code blocks are duplicated. Is there a way to define the set of args once, and re-use for each group that needs it?

Reply:
These were actually duplicated deliberately. The common args were once abstracted out, but as individual subcommands/actions eventually became differentiated the code had become complex and difficult to read. The code was becoming more difficult to read, so we prioritized readabililty. Also separate code blocks for each action allows us to customise further in the future.

Reply:
But the set of 5 lines that define the --fofn arg are 1 "thing", and your intent is that it's the same arg for all the subcommands. So for now if it could be explicit by using a function (to allow for the variable help message) defined once, that would be better: it would be obvious these are supposed to the same thing, if a change is needed it could be changed in one place, and if one of them needs to be different in the future, not using the shared function for that one will make it more obvious it is really supposed to be different.

Reply:
The point about abstracting out common code I understand but I still have a preference for dumb readability over de-duplication. But if you'd like to make the change yourself, feel free